### PR TITLE
feat: init setting

### PR DESCRIPTION
**Pull request by Jian-Min-Huang**
# Checklist:
- [x] My code follows the style guidelines of this project (IntelliJ Default)
- [x] My changes generate no new warnings
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -21,6 +21,21 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-web</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.slack.api</groupId>
+            <artifactId>bolt</artifactId>
+            <version>1.0.0-RC3</version>
```
Comment (jerry80409) [inline pom.xml:27]: 我想請問, 選用 RC3 有特別的目的嗎？
因為我看教學是用 1.0.1 的版本, 目前 Maven 最新也是 1.0.1
image

**Review by jerry80409 — APPROVED** [review]

File: src/main/java/tw/b2e/slack/SlackConfig.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    @Bean
+    public App initSlackApp() {
+        App app = new App();
+        app.command("/stock", (req, ctx) -> ctx.ack(stockRouter.handle(req)));
+        app.command("/news", (req, ctx) -> ctx.ack(newsRouter.handle(req)));
+        app.command("/game", (req, ctx) -> ctx.ack(gameRouter.handle(req)));
+        app.command("/receipt", (req, ctx) -> ctx.ack(receiptRouter.handle(req)));
+        app.command("/drink", (req, ctx) -> ctx.ack(drinkRouter.handle(req)));
+
+        return app;
```
Comment (jerry80409) [inline src/main/java/tw/b2e/slack/SlackConfig.java:37]: 我有稍微整理一下, 但目前的寫法也很 OK :)
```java
Map.ofEntries(
  Map.entry("/stock",   stockRouter),
  Map.entry("/news",    newsRouter),
  Map.entry("/game",    gameRouter),
  Map.entry("/receipt", receiptRouter),
  Map.entry("/drink",   drinkRouter)
).forEach((command, router) -> 
  app.command(command, (req, ctx) -> ctx.ack(router.handle(req))));
```

File: pom.xml (hunk 1 above)
Comment (Jian-Min-Huang) [reply pom.xml:27]: 好像是最新的就用了 呵呵

File: src/main/java/tw/b2e/slack/SlackConfig.java (hunk 2 above)
Comment (Jian-Min-Huang) [reply src/main/java/tw/b2e/slack/SlackConfig.java:37]: 這樣也很好

**Pull request merged**